addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -1493,7 +1493,8 @@ options.registry.Carousel = options.Class.extend({
      * @override
      */
     start: function () {
-        this.$target.carousel('pause');
+        this.$iframeTarget = this.ownerDocument.defaultView.$(this.$target[0]);
+        this.$iframeTarget.carousel('pause');
```

Comment:
I don't understand those lines, is that a common thing you had to do ? :thinking: I don't understand how `$iframeTarget` is different from `$target `

Reply:
I changed $iframeTarget to $bsTarget which I made available to all snippets.options via include. 😬 There might be a better way to do that, or maybe it's even best to do it locally, I'm really not sure.

I also tried data-bs-no-jquery on the frontend and it did break some features of the frontend, and prevented the edition to start, so I thought that for the scope of fixing this, I wouldn't do the changes here.

Here's what I found though: Disabling bootstrap's jQuery mainly impact website_sale, website_slides and the edition code to handle mega menus and dropdowns. Some of the challenges might be that it's not longer possible to access batches of widgets such as doing `$('.popover').popover('hide')` which I have seen at least once.

Probably not necessary but if needed I can push a branch on odoo-dev with my experiments (I tried adapting the code for the website_sale cart dropdown).